RELEASE/scripts/sl_ascend.ash

```diff
@@ -825,7 +825,7 @@ item[monster] catBurglarHeistDesires()
 	if((item_amount($item[killing jar]) == 0) && ((get_property("gnasirProgress").to_int() & 4) == 4) && in_hardcore())
 		wannaHeists[$monster[banshee librarian]] = $item[killing jar];
 
-	if(!possessEquipment($item[Mega Gem]) && in_hardcore() && (item_amount($item[wet stew]) == 0) && (item_amount($item[wet stunt nut stew]) == 0))
+	if((my_level() >= 11) && !possessEquipment($item[Mega Gem]) && in_hardcore() && (item_amount($item[wet stew]) == 0) && (item_amount($item[wet stunt nut stew]) == 0))
```

Comment:
Why add this? Not that I see any problem with it, since I doubt we're ever going to Whitey's Grove without being level 11 or higher, but why is it necessary, out of curiosity?

Reply:
Oh, if you have `sl_debug` enabled, we log our desired cat burglar heists very aggressively. I wanted to remove some of the mind-numbing repetition from it without disabling that logging entirely.

Reply:
Ah, ok, that makes perfect sense.